controls/datepicker/selection.md

```diff
@@ -28,7 +28,7 @@ To define the current date selection, use the `Date`(`DateTime?`) property. By d
 xmlns:telerik="http://schemas.telerik.com/2022/xaml/maui"
  ```
 
-## Clearing the Selection through UI
+## Clearing the Selection Through UI
```

Comment:
Never capitalize prepositions. See [Capitalization](https://testdocs.telerik.com/style-guide/titles-and-headings#capitalization) for details. 
```suggestion
## Clearing the Selection through UI
```